tests/testhelpers/jobHelpers.go

```diff
@@ -67,7 +66,9 @@ func WaitForJobsToCompleteAndLogStatus(
 		for jobId, oldStatus := range jobs {
 
 			if !IsJobCompleted(oldStatus) {
+				log.Println("getting status")
```

Comment:
What were these prints intended to address? It seems like it might unnecessarily spam the output.